client/src/client.ts

```diff
@@ -67,8 +67,8 @@ export class AngularLanguageClient implements vscode.Disposable {
             document: vscode.TextDocument, range: vscode.Range, context: vscode.CodeActionContext,
             token: vscode.CancellationToken, next: lsp.ProvideCodeActionsSignature) => {
           if (await this.isInAngularProject(document) &&
-              isInsideInlineTemplateRegion(document, range.start) &&
-              isInsideInlineTemplateRegion(document, range.end)) {
+              isInsideComponentDecorator(document, range.start) &&
+              isInsideComponentDecorator(document, range.end)) {
```

Comment:
The function seems to also return for HTML files, but the name only suggests "inside decorator". Wondering if we can rename, but otherwise LGTM.